### ISSUE TITLE
Support for Unity dropped?

### ISSUE THREAD

**Issue by FricoRico** [Quobject/SocketIoClientDotNet#69]
I've successfully incorporated your socket library into a WPF application. Therefore I decided I also wanted to use the library with Unity because it is really fast and easy to use.

Sadly Nugget installing won't work. I've also tried to include any of the DLL's from the WPF package library into Unity. They all give an error based on missing resources for that version.

The README.MD states support for Unity, but I couldn't find the appropriate build. Which one do I need for it to work nicely with Unity's environment?

**Comment by FricoRico** [comment]
I managed to get .NET Framework 3.5 working. Unity 5.2's Mono version partially supports .NET 3.5. Just enough features to make it work, all what's missing is System.Threading. For that I used System.Threading.Unofficial library. This is made specially for Mono .NET 3.5.

It now works like a charm and is way faster than any other existing Socket.IO plugins for Unity. From 45-60ms latency to 0-1ms on a internal connection.
